tools/generate-mocks/generate-mocks.ts

```diff
@@ -491,6 +555,38 @@ export class ${mockClassName} {${propertiesString}${methodsString}}
     componentMetaData.methods.push(method);
   }
 
+  private visitCustomElementsJSON(componentMetaData: ComponentMetaData) {
+    const currentComponentSelector = componentMetaData.selector.replace(/["']/g, '');
+    const component = customElements.components.find(
+      (comp) => comp.tag === currentComponentSelector
+    );
+
+    if (component) {
+      component.props.forEach((prop) => {
+        componentMetaData.properties.push({
+          name: prop.name,
+          type: prop.type,
+          initializer: prop.default?.replace(/["']/g, ''),
```

Comment:
Just out of curiosity - what is going on in line 569? 
![image](https://user-images.githubusercontent.com/17269698/126134259-6d089992-5a88-47a7-b02f-fdbf705dd504.png)

Reply:
We strip `"` and `'` by replacing it with nothing 😃